accounts-db/src/accounts.rs

```diff
@@ -669,6 +669,13 @@ impl Accounts {
         );
         self.accounts_db
             .store_cached_inline_update_index((slot, &accounts_to_store[..]), Some(&transactions));
+        StoredAccountsInfo {
+            count: accounts_to_store.len() as u64,
+            data_bytes: accounts_to_store
+                .iter()
```

Comment:
we iterate these and sum data len here:
https://github.com/solana-labs/solana/blob/897adb271196ba75edd752e0d21696cee8610017/accounts-db/src/accounts_db.rs#L8459-L8461